chromadb/api/__init__.py

```diff
@@ -51,7 +51,7 @@ def list_collections(self) -> Sequence[Collection]:
     def create_collection(
         self,
         name: str,
-        metadata: Optional[CollectionMetadata] = None,
+        metadata: Optional[Metadata] = None,
```

Comment:
This is the main change throughout; as I remember `CollectionMetadata` was earlier deprecated.